### PR TITLE
Adding token-bridge subrepository

### LINKED ISSUE #1: Create monorepo with token-bridge and wsrun

### PR DESCRIPTION
**Pull request by rzadp**
First step in the creation of the monorepository.

- Added `token-bridge` from tag [1.3.0-rc0](https://github.com/poanetwork/token-bridge/tree/1.3.0-rc0)
- Added CircleCI integration

Fixes #1

### CODE REVIEW

File: token-bridge/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```
Comment (akolotov) [inline token-bridge/CODE_OF_CONDUCT.md]: Does it make sense to move the file in the root dir of the repo?

File: token-bridge/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+sudo: required
```
Comment (akolotov) [inline token-bridge/.travis.yml]: Does it make sense to keep this file?

File: token-bridge/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+## Contributing
```
Comment (akolotov) [inline token-bridge/CONTRIBUTING.md]: Does it make sense to move the file in the root dir of the repo?

File: token-bridge/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Thank your for contributing to this project! We welcome collaborators and expect users to follow our [code of conduct](CODE_OF_CONDUCT.md) when submitting code or comments.
+
+1. Fork the repo ( https://github.com/poanetwork/bridge-nodejs/fork ).
```
Comment (akolotov) [inline token-bridge/CONTRIBUTING.md]: Please change the URL to the repo

File: token-bridge/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+   
+* Commits should be one logical change that still allows all tests to pass.  We prefer smaller commits if there could be two levels of logic grouping.  The goal is to provide future contributors (including your future self) the reasoning behind your changes and allow them to cherry-pick, patch or port those changes in isolation to other branches or forks.
+* If during your PR you reveal a pre-existing bug and know how to fix it:
+  1. If you can isolate the bug, fix it in a separate PR.
+  2. If the fix depends on your other commits, add it in a separate commit to the same PR.  
+
+    In either case, try to write a regression test that fails because of the bug but passes with your fix.
+
+### Issues
+Creating and discussing [Issues](https://github.com/poanetwork/bridge-nodejs/issues) provides significant value to the project. If you find a bug you can report it in an Issue.
```
Comment (akolotov) [inline token-bridge/CONTRIBUTING.md]: Please change the URL to the repo

File: token-bridge/LICENSE [hunk 6]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (akolotov) [inline token-bridge/LICENSE]: Does it make sense to move this file to the root dir of the repo?

File: token-bridge/README.md [hunk 7]
```diff
@@ -0,0 +1,297 @@
+
+# POA TokenBridge
+
+[![Build Status](https://travis-ci.org/poanetwork/token-bridge.svg)](https://travis-ci.org/poanetwork/token-bridge)
```
Comment (akolotov) [inline token-bridge/README.md]: Does it make sense to remove the TravisCI badge and move the gitter badge to the README file in the root directory of the repo?

File: token-bridge/README.md [hunk 8]
```diff
@@ -0,0 +1,297 @@
...
+| `HOME_BRIDGE_ADDRESS` | Address of the bridge in the Home network to send transactions. |
+| `HOME_MIN_AMOUNT_PER_TX` | Value (in _eth_ or tokens) to be sent in one transaction for the Home network. This should be greater than or equal to the value specified in the `poa-bridge-contracts/deploy/.env` file. The default value in that file is 500000000000000000, which is equivalent to 0.5. |
+| `HOME_TEST_TX_GAS_PRICE` | The gas price (in Wei) that is used to send transactions in the Home network . |
+| `FOREIGN_BRIDGE_ADDRESS` | Address of the bridge in the Foreign network to send transactions. |
+| `FOREIGN_MIN_AMOUNT_PER_TX` | Value (in _eth_ or tokens) to be sent in one transaction for the Foreign network. This should be greater than or equal to the value specified in the `poa-bridge-contracts/deploy/.env` file. The default value in that file is 500000000000000000, which is equivalent to 0.5. |
+| `FOREIGN_TEST_TX_GAS_PRICE` | The gas price (in Wei) that is used to send transactions in the Foreign network . |
+
+## Contributing
+
+See the [CONTRIBUTING](CONTRIBUTING.md) document for contribution, testing and pull request protocol.
```
Comment (akolotov) [inline token-bridge/README.md]: Either the link to CONTRIBUTING.md needs to be corrected or this sections should be moved to the README in the root dir of the repo

File: token-bridge/README.md [hunk 9]
```diff
@@ -0,0 +1,297 @@
...
+
+## License
+
+[![License: LGPL v3.0](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+
+This project is licensed under the GNU Lesser General Public License v3.0. See the [LICENSE](LICENSE) file for details.
```
Comment (akolotov) [inline token-bridge/README.md]: Should the link to the LICENSE file be changed?

**Comment by rzadp** [comment]
@akolotov My idea was to introduce completely unchanged code from 1.3.0-rc0 (and later unchanged other sub-projects), so that any changes done to sub-projects can be reviewed and referenced separately.

How do you feel about doing your requested changes in a subsequent PR and working in small steps in general?

**Comment by akolotov** [comment]
> so that any changes done to sub-projects can be reviewed and referenced separately.

Oh! I see! Just did not find issues planned in the project to cover this work that's why decided that it should be finished here.

So, my proposal is to create an issue for this work and I will move my remarks there. Is it OK for you?

**Comment by rzadp** [comment]
@akolotov I went ahead and created the issue with remarks as a checklist: #17

**Comment by akolotov** [comment]
> @akolotov I went ahead and created the issue with remarks as a checklist: #17

great thanks!

File: token-bridge/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (akolotov) [reply token-bridge/CODE_OF_CONDUCT.md]: Addressed in https://github.com/poanetwork/tokenbridge/pull/19

File: token-bridge/.travis.yml (hunk 2 above)
Comment (akolotov) [reply token-bridge/.travis.yml]: Addressed by https://github.com/poanetwork/tokenbridge/pull/19

File: token-bridge/CONTRIBUTING.md (hunk 3 above)
Comment (akolotov) [reply token-bridge/CONTRIBUTING.md]: Addressed by https://github.com/poanetwork/tokenbridge/pull/19

File: token-bridge/CONTRIBUTING.md (hunk 4 above)
Comment (akolotov) [reply token-bridge/CONTRIBUTING.md]: Addressed by https://github.com/poanetwork/tokenbridge/pull/19

File: token-bridge/CONTRIBUTING.md (hunk 5 above)
Comment (akolotov) [reply token-bridge/CONTRIBUTING.md]: Addressed by https://github.com/poanetwork/tokenbridge/pull/19

File: token-bridge/LICENSE (hunk 6 above)
Comment (akolotov) [reply token-bridge/LICENSE]: Addressed by https://github.com/poanetwork/tokenbridge/pull/19

File: token-bridge/README.md (hunk 7 above)
Comment (akolotov) [reply token-bridge/README.md]: Addressed by https://github.com/poanetwork/tokenbridge/pull/19

File: token-bridge/README.md (hunk 8 above)
Comment (akolotov) [reply token-bridge/README.md]: Addressed by https://github.com/poanetwork/tokenbridge/pull/19

File: token-bridge/README.md (hunk 9 above)
Comment (akolotov) [reply token-bridge/README.md]: Addressed by https://github.com/poanetwork/tokenbridge/pull/19

**Pull request merged**